docs/resources/user.md

```diff
@@ -43,26 +43,879 @@ resource "snowflake_user" "user" {
 
 ### Optional
 
+- `abort_detached_query` (Boolean) Specifies the action that Snowflake performs for in-progress queries if connectivity is lost due to abrupt termination of a session (e.g. network outage, browser termination, service interruption). For more information, check [ABORT_DETACHED_QUERY docs](https://docs.snowflake.com/en/sql-reference/parameters#abort-detached-query).
```

Comment:
Add custom template for user with the V1 candidate message

Reply:
It's not yet the rework of the whole user resource (this implementation was there before).

Changed nevertheless (had to leave the condition to avoid different behavior before the redesign).